lib/Chat/Parser/SystemMessage.php

```diff
@@ -221,6 +259,11 @@ public function parseMessage(Message $chatMessage): void {
 				$parsedMessage = $this->l->t('You promoted {user} to moderator');
 			} elseif ($participant->isGuest() && $participant->getSessionId() === $parsedParameters['user']['id']) {
 				$parsedMessage = $this->l->t('{actor} promoted you to moderator');
+				if ($cliIsActor) {
+					$parsedMessage = $this->l->t('An administrator promoted you to moderator');
+				}
+			} elseif ($cliIsActor) {
+				$parsedMessage = $this->l->t('An administrator promoted {user} to moderator');
```

Comment:
`guest_moderator_promoted` is never reached since we can't promote guests to moderators via CLI.

Reply:
well we can't so far, I'd like to keep it in case we do it at some point.